srcpkgs/pinentry/template

```diff
@@ -18,6 +19,9 @@ checksum=68076686fa724a290ea49cdf0d1c0c1500907d1b759a3bcbfbec0293e8f56570
 
 alternatives="pinentry:pinentry:/usr/bin/pinentry-curses"
 
+build_options="libsecret"
+build_options_default="libsecret"
```

Comment:
libsecret has no entry on common/shlibs, please add

desc_option_libsecret=""

fill it with a very simple explanation of what it is.

grep for `desc_option_` for examples or look into `common/options.description`